modules/plotting/include/modules/plotting/properties/optionconverter.h

```diff
@@ -44,7 +44,10 @@ class DataFrameColumnToOptionConverter
 protected:
     virtual void convertimpl(const plot::DataFrameColumnProperty *src,
                              OptionProperty *dst) const override {
-        dst->setSelectedIndex(src->getSelectedIndex());
+        if (src->getSelectedIndex() > dst->getValues().size())
```

Comment:
`dst->setSelectedIndex(glm::clamp(0, dst->size()-1, src->getSelectedIndex())`

Reply:
Nice! But I get an error when trying it:
![image](https://user-images.githubusercontent.com/8807821/53242582-76137080-36a5-11e9-852f-8bb133c269f1.png)
Both for giving glm::clamp the template argument dst->size() and also src->getSelectedIndex()

Reply:
That was just from the top of my head :) I guess clamp is not needed since it is size_t. How about something like
`dst->setSelectedIndex(std::min(src->getSelectedIndex(), dst->size() - 1));`

Reply:
Try
dst->setSelectedIndex(glm::clamp<size_t>(0, dst->size()-1, src->getSelectedIndex())